recipes-kernel/linux/linux-remarkable2_5.4.bb

```diff
@@ -0,0 +1,14 @@
+require recipes-kernel/linux/linux-imx.inc
+
+SUMMARY = "Linux kernel for the reMarkable2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=bbea815ee2795b2f4230826c0c6b8814"
+
+SRC_URI = "git://github.com/alistair23/linux.git;protocol=git;branch=${SRCBRANCH}"
+SRCBRANCH = "5.4-rM2-2.2.x-imx"
```

Comment:
I think you should use `KBRANCH`.

Reply:
I don't see any other recipes using `KBRANCH`, I'm happy to change it though, it doesn't matter as long as it matches the `branch=` in `SRC_URI`

Reply:
An example: https://github.com/Freescale/meta-freescale/commit/ea0559e137de4823745baf1ef2bca8af972564aa